cypress/integration/view/offline/offline.js

```diff
@@ -175,10 +175,12 @@ Given('I open an uncached dashboard', () => {
 
 When('connectivity is turned off', () => {
     goOffline()
+    cy.wait(500) // eslint-disable-line cypress/no-unnecessary-waiting
```

Comment:
Any way to get around the `wait`?

Reply:
Could potentially wait until one of the action buttons becomes enabled. I can try that out.